### PR TITLE
Javascript disabled

### PR DESCRIPTION
**Pull request by dethos**
Handle the situations when the JavaScript it disabled on the browser.

This is an attempt to fix the following issue: https://github.com/whitesmith/hawkpost/issues/33

### CODE REVIEW

**Comment by nunosilva800** [comment]
LGTM!

File: boxes/static/javascripts/box_submit.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    $form.append($csrfTokenField);
+    $form.append($("<p></p>").append($label).append($textArea));
+    $form.append($("<a id='encrypt-action-js' class='btn-default'>Encrypt and Send</a>"));
+    $formDiv.append($form);
+
+    $("#encrypt-action-js").on("click", encryptContent);
+  }
+
+  createBox();
```
Comment (ricardopereira) [inline boxes/static/javascripts/box_submit.js:66]: The form disappear when the browser doesn't run js?

**Comment by ricardopereira** [comment]
LGTM. Just a small comment (trying to learn something because I'm a web noobie 😅)

File: boxes/static/javascripts/box_submit.js (hunk 1 above)
Comment (nunosilva800) [inline boxes/static/javascripts/box_submit.js:66]: Actually: the form appears if browser supports JS. :)

**Comment by dpflucas** [comment]
👍

**Pull request merged**